include/ofi_mem.h

```diff
@@ -273,6 +274,7 @@ struct util_buf_attr {
 	size_t 				chunk_cnt;
 	util_buf_region_alloc_hndlr 	alloc_hndlr;
 	util_buf_region_free_hndlr 	free_hndlr;
+	util_buf_region_unavail_hndlr	unavail_hndlr;
```

Comment:
We already have a alloc_hndlr.  This looks like a pre_alloc_handler.  In many cases, the user could perform the necessary actions as part of alloc_handler.  The pool would grow first, but I'm not sure that really matters.